src/pages/questions/questions.ts

```diff
@@ -147,7 +147,7 @@ export class QuestionsPage {
     const percent = Math.ceil(
       (this.currentQuestion * 100) / this.questions.length
     )
-    this.progress = percent + tick
+    this.progress = percent
```

Comment:
Why was the tick used here ? I am a bit confused

Reply:
I am also not sure why it was used. But I've removed it.